### PR TITLE
refactor: Unify ingress object into one, expose API and UI under same URL

### LINKED ISSUE #15: Define Ingress for /webhook

### PR DESCRIPTION
**Pull request by mjasion**
Already we are defining separate ingress configs for UI and API. The scenario, when a user would like to expose the studio API on different endpoints looks to be rare. Also, https://studio.iterative.ai runs on the same host. 

This PR just makes easier to configure Studio including ingress object.

Also add `/webhook` endpoint. Fixes #15

### CODE REVIEW

File: charts/studio/templates/configmap-studio.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: studio
 data:
   ALLOWED_HOSTS: "*"
-  API_URL:  "http{{ if $.Values.studioBackend.ingress.tlsEnabled }}s{{ end }}://{{ .Values.studioBackend.ingress.host }}/"
-  UI_URL: "http{{ if $.Values.studioUi.ingress.tlsEnabled }}s{{ end }}://{{ .Values.studioUi.ingress.host }}/"
+  API_URL: "{{.Values.global.ingress.host }}/api"
```
Comment (mjasion) [inline charts/studio/templates/configmap-studio.yaml:7]: I am not pretty sure if API_URL should contain `/api` path. In studio code, and in studio-selfhosted I see that it is set as default:
https://github.com/iterative/studio-selfhosted/blob/ca74133612604abb22e449810574690f4bb1bf55/docker-compose/base.yaml#L62

File: charts/studio/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            backend:
+              {{- if semverCompare ">=1.19-0" $.Capabilities.KubeVersion.GitVersion }}
+              service:
+                name: studio-backend
+                port:
+                  number: {{ .Values.studioBackend.service.port }}
+              {{- else }}
+              serviceName: studio-backend
+              servicePort: {{ .Values.studioBackend.service.port }}
+              {{- end }}
```
Comment (jesper7) [inline charts/studio/templates/ingress.yaml:79]: Dupe

**Review by jesper7 — CHANGES_REQUESTED** [review]
Nice! :slightly_smiling_face:

File: charts/studio/templates/configmap-studio.yaml (hunk 1 above)
Comment (jesper7) [reply charts/studio/templates/configmap-studio.yaml]: It's correct :+1:

File: charts/studio/templates/ingress.yaml (hunk 2 above)
Comment (mjasion) [reply charts/studio/templates/ingress.yaml]: Færdig

File: charts/studio/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{{- if .Values.global.ingress.enabled -}}
```
Comment (omesser) [inline charts/studio/templates/ingress.yaml:1]: my 2 cents - better to have 2 ingresses - 1 for each service. it's less confusing and easy to reuse the values/vars needed
If you still go with this - still rename file to include resource name `ingress-studio.yaml` please

**Review by omesser — CHANGES_REQUESTED** [review]

File: charts/studio/templates/ingress.yaml (hunk 3 above)
Comment (jesper7) [reply charts/studio/templates/ingress.yaml:1]: It's all the same service to me, but it depends on how you look at it. 
In this case, I like having everything in one Ingress.
But agree with the renaming suggestion.

**Review by omesser — APPROVED** [review]

**Review by jesper7 — APPROVED** [review]

**Pull request merged**